CHANGES.rst

```diff
@@ -1,16 +1,21 @@
 Changes
 =======
 
-Version 0.7.3 (UNRELEASED)
+Version 0.7.3 (2021-03-17)
 --------------------------
 
+- Users:
+    - Adds ``reana-client validate`` options to detect possible issues with workflow input parameters and environment images.
+    - Fixes problem with failed jobs being reported as still running in case of network problems.
+    - Fixes job command encoding issues when dispatching jobs to HTCondor and Slurm backends.
+    - Changes job status consumer communications to improve overall platform resilience.
```

Comment:
The last news, status consumer communications, is more for Administrators than for Users... even though it is the users that benefit at the end.  

(See also 'improves platform resilience` news item that we have in Administrators section, it could be joined with this one or else expanded how the resilience is improved...)